src/main/liberty/config/server.xml

```diff
@@ -2,21 +2,33 @@
 
   <featureManager>
       <feature>microprofile-1.3</feature>
+      <feature>usr:opentracingZipkin-0.30</feature>
   </featureManager>
 
   <httpEndpoint httpPort="${default.http.port}" httpsPort="${default.https.port}"
       id="defaultHttpEndpoint" host="*" />
 
   <!-- Automatically expand WAR files and EAR files -->
   <applicationManager autoExpand="true"/>
+    
+  <opentracingZipkin host="${env.zipkinHost}" port="${env.zipkinPort}"/>
+  
+  <webApplication name="customer" location="${app.location}">
+      <classloader apiTypeVisibility="api,ibm-api,spec,stable,third-party"/>
+  </webApplication>
```

Comment:
I don't think you need this
`<classloader apiTypeVisibility="api,ibm-api,spec,stable,third-party"/>`

Seems like you are not using the custom tracer anywhere.